docs/source/index.mdx

```diff
@@ -15,15 +15,36 @@ limitations under the License.
 -->
 
 
-# Optimum for Intel Gaudi
+# Optimum for Intel® Gaudi® AI Accelerator
 
-Optimum for Intel Gaudi is the interface between the Transformers and Diffusers libraries and [Intel® Gaudi® AI Accelerators (HPUs)](https://docs.habana.ai/en/latest/index.html).
+Optimum for Intel Gaudi AI accelerator is the interface between Hugging Face libraries (Transformers, Diffusers, Accelerate,...) and [Intel Gaudi AI Accelerators (HPUs)](https://docs.habana.ai/en/latest/index.html).
 It provides a set of tools that enable easy model loading, training and inference on single- and multi-HPU settings for various downstream tasks as shown in the table below.
 
-HPUs offer fast model training and inference as well as a great price-performance ratio.
-Check out [this blog post about BERT pre-training](https://huggingface.co/blog/pretraining-bert) and [this post benchmarking Intel Gaudi 2 with NVIDIA A100 GPUs](https://huggingface.co/blog/habana-gaudi-2-benchmark) for concrete examples.
-If you are not familiar with HPUs, we recommend you take a look at [our conceptual guide](./concept_guides/hpu).
+<div class="mt-10">
+  <div class="w-full flex flex-col space-y-4 md:space-y-0 md:grid md:grid-cols-2 md:gap-y-4 md:gap-x-5">
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="./tutorials/overview"
+      ><div class="w-full text-center bg-gradient-to-br from-blue-400 to-blue-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Tutorials</div>
+      <p class="text-gray-700">Learn the basics and become familiar with training transformers on HPUs with 🤗 Optimum. Start here if you are using 🤗 Optimum for Intel Gaudi for the first time!</p>
+    </a>
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="./usage_guides/overview"
+      ><div class="w-full text-center bg-gradient-to-br from-indigo-400 to-indigo-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">How-to guides</div>
+      <p class="text-gray-700">Practical guides to help you achieve a specific goal. Take a look at these guides to learn how to use 🤗 Optimum for Intel Gaudi to solve real-world problems.</p>
+    </a>
+  </div>
+</div>
+
+The Intel Gaudi AI accelerator family currently includes three product generations:
+[Intel Gaudi 1](https://habana.ai/training/gaudi),
+[Intel Gaudi 2](https://habana.ai/training/gaudi2), and
+[Intel Gaudi 3](https://habana.ai/training/gaudi3).
```

Comment:
```suggestion
[Intel Gaudi 3](https://habana.ai/products/gaudi3/).
```
https://habana.ai/training/gaudi3 doesn't exist anymore it seems. Actually even https://habana.ai/training/gaudi and https://habana.ai/training/gaudi2 redirects to https://habana.ai/products/gaudi and https://habana.ai/products/gaudi2.

Reply:
Wow this works few days ago!  Ill update now

Reply:
Fixed